src/parser/jobs/ast/modules/ArcanaTracking/ArcanaSuggestions.tsx

```diff
@@ -200,7 +212,7 @@ export default class ArcanaSuggestions extends Module {
 
 	// Helper for output()
 	RenderSpreadState(artifact: CardLog) {
-		const drawnArcana = getDataBy(STATUSES, 'id', artifact.drawState)
+		const drawnArcana = artifact.drawState ? this.data.getStatus(artifact.drawState) : undefined
```

Comment:
Apologies if this was in before and I missed it, but I also noticed that the Arcana Logs are no longer showing the placeholder in Spread State for when there is no active card, instead showing a red square with an X:
![image](https://user-images.githubusercontent.com/26804378/80893339-7a6eec80-8c9f-11ea-880d-ecad657c29e0.png)

Reply:
This PR actually fixes this. The problem was, we used to use "0" as inactive state, but it turns out that "0" represented the "X" icon.
So now we're using undefined if there's no active card, which means it should render the placeholder.

Reply:
Ah, whoops, I got my windows crossed up 😊